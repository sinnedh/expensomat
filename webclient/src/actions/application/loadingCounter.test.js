import * as actions from './loadingCounter'

describe('APPLICATION', () => {
  it('handles resetLoadingCounter', () => {
    const action = actions.resetLoadingCounter()
    expect(action).toMatchSnapshot()
  })

  it('handles incrementLoadingCounter', () => {
    const action = actions.incrementLoadingCounter()
    expect(action).toMatchSnapshot()
  })

  it('handles decrementLoadingCounter', () => {
    const action = actions.decrementLoadingCounter()
    expect(action).toMatchSnapshot()
  })
})
