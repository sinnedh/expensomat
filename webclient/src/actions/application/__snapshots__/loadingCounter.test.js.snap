// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APPLICATION handles decrementLoadingCounter 1`] = `
Object {
  "type": "APPLICATION:DECREMEMT_LOADING_COUNTER",
}
`;

exports[`APPLICATION handles incrementLoadingCounter 1`] = `
Object {
  "type": "APPLICATION:INCREMEMT_LOADING_COUNTER",
}
`;

exports[`APPLICATION handles resetLoadingCounter 1`] = `
Object {
  "type": "APPLICATION:RESET_LOADING_COUNTER",
}
`;
